packages/docusaurus-plugin-openapi-docs/src/markdown/createAuthentication.ts

```diff
@@ -167,11 +167,16 @@ export function createAuthentication(securitySchemes: SecuritySchemeObject) {
 
   return create("div", {
     children: [
-      create("h2", {
-        children: "Authentication",
-        id: "authentication",
-        style: { marginBottom: "1rem" },
-      }),
+      create(
+        "Heading",
```

Comment:
I tried using nested `Markdown` here with children as `## Authentication` but for some reason the component renders an object as children and fails to render children. I guess the other cases where "Markdown" are children of "TabItem" are limited to some sort of "TabItem" custom implementation

Reply:
Looks good except we seem to be missing the `marginBottom` from before. Wondering if this deserves a new class or if there's a way to add the margin to `openapi-tabs__heading` without disrupting other components.

Reply:
> I added `openapi-tabs__heading` class as a different one from other nested tabs headings so I think we can add the margin without disruptions

Had to add an additional class to `ApiTabs` margin case